src/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <script src="https://kit.fontawesome.com/a0887fcaf6.js" crossorigin="anonymous"></script>
+
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do List</title>
+    <script src="https://unpkg.com/lodash@4.17.20"></script>
+    <script src="./index.js" defer></script>
+</head>
+<body>
+    <main>
+        <section id="hero">
+            <h1><span>Today's To Do</span> <i class="fa-solid fa-recycle"></i></h1>
+            <input id="add-list" type="text" placeholder="Add to your list..." required>
+            <ul id="to-do-list">
+            </ul>
+            <div id="reset" class="pointer">Clear All Completed</div>
+        </section>
+    </main>
```

Comment:
Kindly implement the Add button function in your code and also add borders or background color to your input, for a better user experience